data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg

```diff
@@ -206,6 +206,13 @@
         {FLAG_VARIANT knalgan}
         team_name=dwarves
         user_team_name= _ "Dwarves"
+        # When I first edited this scenario I didn't realize that
```

Comment:
I just played S4, I'm pretty sure these changes aren't really in the direction I want to go with this scenario. I think I'd like to keep the lower turn limit and simply give you more gold if necessary on easier difficulties. How about I handle this scenario in my rebalance pass? S6 looks okay to me though.

Reply:
ok... so do you want me to revert the S4 changes then? Or can we keep them and just have you overwrite them when you do your own rebalance pass?

Reply:
I think it doesn't make sense to commit them to begin with if I'm going to overwrite them anyway so I think it's better to not include them in this PR.

Reply:
> I think it doesn't make sense to commit them to begin with if I'm going to overwrite them anyway so I think it's better to not include them in this PR.

ok reverted in c9f8a61